src/python/tests/core/bot/fuzzers/engine_common_test.py

```diff
@@ -459,3 +459,22 @@ def test_unpack_seed_corpus_subdirs(self):
     self._write_seed_corpus(self.seed_corpus_subdirs_contents, '.zip')
     self._unpack_seed_corpus_if_needed()
     self._assert_elements_equal(expected_dir_contents, self._list_corpus_dir())
+
+
+class GetRadamsaOutputFilenameTest(unittest.TestCase):
+  """get_radamsa_output_filename tests."""
+
+  def test_get_output_filename(self):
+    output_filename = engine_common.get_radamsa_output_filename('file', 0)
+    self.assertEqual('radamsa-00001-file', output_filename)
+
+  def test_no_double_prefix(self):
+    output_filename = engine_common.get_radamsa_output_filename(
+        'radamsa-00002-file', 0)
+    self.assertEqual('radamsa-00001-file', output_filename)
+
```

Comment:
can we add a test for existing files with multiple radamsa prefixes like

radamsa-00001-radamsa-00002-

Reply:
I don't strip all radamsa prefixes if there are multiple. But because of the way we are writing them we should never have more than one anyway.